### DISCUSSION TITLE
How to use the multiplexer function in tabby?

### DISCUSSION THREAD

**Discussion by davixd** [Eugeny/tabby#5516] [Q&A] [answered]
Hi guys,
how can I use the new multiplexer function in tabby? My goal is to open several tabs or split screens and to control all of them through as master terminal.

Thank you very much for help!

Best regards
David

**Comment by Eugeny** [comment, marked as answer]
If you're referring to typing into multiple sessions at once, you can do it by splitting a tab and then using the "Focus all panes" hotkey or context menu item.

If you mean SSH session multiplexing (where you reuse a single connection for multiple tabs), you can enable it in the connection's advanced settings.

**Reply by davixd** [reply]
Thank Eugeny, it did worked as described.

**Comment by lawweiliang** [comment]
@davixd   
Please close the discussion if the question was answered. 
It would be helpful to make the discussion section clean. 

Click `mark as answered` and `close the discussion`. 😄 

![image](https://user-images.githubusercontent.com/34936929/228098333-222ac3d4-4c39-4c26-a2a8-52906647e00e.png)
